### ISSUE TITLE
Adjust @Generated import line to jakarta package

### ISSUE THREAD

**Issue by martinariehm** [helospark/SparkBuilderGenerator#66]
When creating `@Generated`-Annotations SparkBuilderGenerator adds an import line
```
import javax.annotation.Generated;
```
However, since the release of Jakarta EE9 should be changed to
```
import jakarta.annotation.Generated;
```
For compatibility reasons, maybe a configuration option to choose the package name should be implemented,

**Comment by helospark** [comment]
Thanks for raising this @martinariehm .
I have released the change in 0.0.27 version, please update and try.
You need to enable the `Use new jakarta.annotation package` in the preferences (preferences->Java->Spark builder generator)

**Comment by martinariehm** [comment]
> Thanks for raising this @martinariehm . I have released the change in 0.0.27 version, please update and try. You need to enable the `Use new jakarta.annotation package` in the preferences (preferences->Java->Spark builder generator)

Confirmed, it works. Thanks a lot!
